src/actions/pause.js

```diff
@@ -225,3 +235,35 @@ export function loadObjectProperties(object: any) {
     });
   };
 }
+
+/**
+ * @memberOf actions/pause
+ * @static
+ * @param stepType
+ * @returns {function(ThunkArgs)}
+ */
+export function astCommand(stepType: string) {
+  return async ({ dispatch, getState, sourceMaps }: ThunkArgs) => {
+    if (!features.asyncStepping) {
+      return dispatch(command({ type: stepType }));
+    }
+
+    const pauseInfo = getPause(getState());
+    const source = getSelectedSource(getState()).toJS();
+    const currentHiddenBreakpointLocation = getHiddenBreakpointLocation(
+      getState()
+    );
+    if (currentHiddenBreakpointLocation) {
```

Comment:
getHiddenBreakpointLocation seems to get the first hidden breakpoint, do we ever have a case of more than one hidden breakpoint? could this give us a false positive?